### PR TITLE
Updates for chapter 2 on 5490

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [1663](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1663) and [1660](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1660)

screen shot 2017-03-16 at 2 26 13 pm

### CODE REVIEW

**Comment by goldenmeanie** [comment]
LGTM!

**Comment by goldenmeanie** [comment]
Oops! Should have the conditional group format.

File: src/js/common/schemaform/widgets/RadioWidget.jsx [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
   disabled,
   onChange,
   id
 }) {
-  const { enumOptions, labels = {} } = options;
+  const { enumOptions, labels = {}, nestedContent = {} } = options;
+
+  // nested content could be a component or just jsx/text
+  let content = nestedContent[value];
+  if (typeof content === 'function') {
```
Comment (cvalarida) [inline src/js/common/schemaform/widgets/RadioWidget.jsx]: Is this the best way to test if a variable is a React component? I haven't found a better one... :confused:

Comment (jbalboni) [inline src/js/common/schemaform/widgets/RadioWidget.jsx]: As far as I know, it is.

File: docs/schemaform/form-config.md [hunk 2]
```diff
@@ -196,6 +196,13 @@ In addition to the uiSchema options listed in the library [https://github.com/mo
     labels: {
       chapter30: 'A readable description (Chapter 30)'
     },
+
+    // This is a map of values to a component, some text, or some jsx. If your field
+    // is a radio widget, the content here will be shown underneath the radio button
+    // for that value when it is selected.
+    nestedContent: {
```
Comment (jbalboni) [inline docs/schemaform/form-config.md]: This is only for radio buttons. They're a single field/widget, so the only way to put content in between the options is some kind of option the widget can read.

Comment (jbalboni) [inline docs/schemaform/form-config.md]: I'm open to suggestions on how to unify those different approaches. The `view:descriptionBox` with an empty schema thing we've been doing feels hackish, so I'd love to get rid of it, but I don't have a better approach off hand for what we're using it for off hand.

Comment (cvalarida) [inline docs/schemaform/form-config.md]: Makes sense. I think we should have a blurb in the docs defining the differences between the different ways to expand / hide content, but that could be in a different PR.

**Comment by cvalarida** [comment]
@jbalboni When this gets merged, could you check off the appropriate item in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1683?
